src/file.ts

```diff
@@ -361,7 +361,9 @@ export interface CopyOptions {
   contentType?: string;
   contentDisposition?: string;
   destinationKmsKeyName?: string;
-  metadata?: FileMetadata;
+  metadata?: {
+    [key: string]: string;
```

Comment:
Apologies, realized this after I made some similar changes yesterday. We probably need to expand the types of this, similar to: https://github.com/googleapis/nodejs-storage/pull/2442/files#diff-bc9705d0f7a567399044dfc66ccc82d4d9aa1cff116842a0094d54e463c9ecbcR473